etna/datasets/internal_datasets.py

```diff
@@ -301,7 +301,7 @@ def read_data(path: Path, part: str) -> np.ndarray:
         with open(path, "r") as f:
             if part in ("randperm", "stations_list"):
                 data = f.read().lstrip("[").rstrip("]\n").split(" ")
-                out = np.array(map(int, data)) if part == "randperm" else np.array(data)
+                out = np.array(list(map(int, data))) if part == "randperm" else np.array(data)
```

Comment:
Does this fix anything?

Reply:
Yes, problem was with this line. I checked it locally.